src/requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+-r requirements.txt
+
+pytest
+pytest-asyncio                        
+pytest-cov                            
+pytest-flakefinder                     
+pytest-forked                          
+pytest-mock                            
+pytest-randomly                        
+pytest-timeout                         
+pytest-xdist
+
+asynctest
+coverage
```

Comment:
Why is `coverage` required?

Reply:
If it is `requirements.txt` for developers, it makes sense.
Have changed requirements to `requirements.txt` for tests only.